apps/asn1pars.c

```diff
@@ -27,27 +27,32 @@ typedef enum OPTION_choice {
 } OPTION_CHOICE;
 
 const OPTIONS asn1parse_options[] = {
+    OPT_SECTION("General"),
     {"help", OPT_HELP, '-', "Display this summary"},
+    {"oid", OPT_OID, '<', "file of extra oid definitions"},
+
+    OPT_SECTION("I/O"),
```

Comment:
In others, you have separated Input from Output.  Not here, is there a reason?

Reply:
see https://github.com/openssl/openssl/pull/9953#issuecomment-539992422